### ISSUE TITLE
README.md includes undocumented array key "card"

### ISSUE THREAD

**Issue by salcode** [stripe/stripe-php#321]
Under [Getting Started](https://github.com/stripe/stripe-php/blob/master/README.md#getting-started) in `README.md`  the following line appears

```
$charge = \Stripe\Charge::create(array('card' => $myCard, 'amount' => 2000, 'currency' => 'usd'));
```

The documentation at https://stripe.com/docs/api#create_charge does not mention a `card` parameter.

Should this perhaps be changed to `source`?

**Comment by brandur** [comment]
Good point! Fixed in 0eaf109. I also modernized the format of the API key and removed the part about sending credit card information so as to not encourage handling card details on the server side.

Thanks!

**Comment by salcode** [comment]
Awesome, thanks for the quick response.  Cheers.
